src/palace/manager/sqlalchemy/model/coverage.py

```diff
@@ -312,6 +315,48 @@ def to_data(self):
             counter=self.counter,
         )
 
+    @property
```

Comment:
These are just helpers to make working with `Timestamp` easier. They could come out to a separate PR if desired.